src/sentry/static/sentry/app/data/forms/sentryApplication.jsx

```diff
@@ -6,6 +6,12 @@ const forms = [
     // Form "section"/"panel"
     title: 'Integration Details',
     fields: [
+      {
+        name: 'internal',
+        label: 'Internal',
+        type: 'boolean',
+        help: 'Choose this to make your integration Internal.',
```

Comment:
What does `internal` mean?

Reply:
tried to add better wording but I do think we'll have to link to docs to explain it more. 
tl;dr is there is no oauth process, they just get an access token, and there is no installing or uninstalling. they can just use the token (and regenerate it if need be)

Reply:
Maybe it'd be more descriptive as "Skips OAuth and only accessible in this Org" or something concise that describes what you did above?

Reply:
personally, I think saying it's automatically installed says more about whats happening than skips OAuth.